java-client/src/main/java/org/opensearch/client/opensearch/core/SearchRequest.java

```diff
@@ -902,6 +915,12 @@ protected void serializeInternal(JsonGenerator generator, JsonpMapper mapper) {
 			generator.write(this.minScore);
 
 		}
+
+		if (this.pit != null) {
```

Comment:
Looking at https://github.com/opensearch-project/opensearch-py/pull/191, should this be called `point_in_time`?

Reply:
So, the JSON object key here is `pit` -- https://opensearch.org/docs/latest/search-plugins/point-in-time-api/#extend-a-pit-time.

Similarly, its `pits` and `pit_id` in different places -- https://opensearch.org/docs/latest/search-plugins/point-in-time-api/#list-all-pits

Reply:
Ah ... it seems like the conventions are not consistent, fe https://github.com/opensearch-project/opensearch-rs/pull/157/files uses `pit` to refer to point-in-time feature (and related APIs/models)

Reply:
Right, I followed the API specs when it came to naming the object variables.